ts/features/bonus/common/components/BonusInformationComponent.tsx

```diff
@@ -43,6 +43,7 @@ type OwnProps = {
   onConfirm?: () => void;
   onCancel?: () => void;
   primaryCtaText: string;
+  secondaryAction?: { type: "back"; text: string };
```

Comment:
What do you think to create a custom type about this secondaryAction property? Since it has more than one attribute, we usually create a custom-type

Reply:
I personally prefer to inline as much code as possible to not have to find naming and indirection where reading code. I'll do